cmd/manager/main.go

```diff
@@ -156,10 +158,31 @@ func parseFlags() {
 	flag.BoolVar(&enableHRQ, "enable-hrq", false, "Enables hierarchical resource quotas")
 	flag.StringVar(&hncNamespace, "namespace", "hnc-system", "Namespace where hnc-manager and hnc resources deployed")
 	flag.DurationVar(&hrqSyncInterval, "hrq-sync-interval", 1*time.Minute, "Frequency to double-check that all HRQ usages are up-to-date (shouldn't be needed)")
+	flag.Var(&excludeResourcesWithLabel, "exclude-objects-with-label", "An annotation specified as key=val that, if present, will cause HNC to skip objects that match this label. May be specified multiple times, with each instance specifying one label. See the user guide for more information.")
```

Comment:
"An annotation" -> "A label"

"with each instance specifying one label" -> "one key/value pair" (or something like that)

Maybe change the name to `--nopropagation-label` to match the terminology [here](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/how-to.md#limit-the-propagation-of-an-object-to-descendant-namespaces)? And ditto for the internal names.

This will also make it more natural to document - e.g. "you can use these built-in annotations to disable propagation, _or_ if the objects already have the labels on them (e.g. because Rancher set them), the HNC admin can set this command-line arg."